.github/workflows/bindgen.yml

```diff
@@ -48,11 +48,11 @@ jobs:
           profile: minimal
           # MSRV below is documented in README.md, please update that if you
           # change this.
```

Comment:
If you make the `rust-version` change, then maybe we should update this comment to point at `Cargo.toml`, and comment the addition in `Cargo.toml` to point here.